spec/features/merchant_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Merchant do
+  #6. Merchant Items Index Page
+  describe "As a merchant," do
+    describe "When I visit my merchant items index page (merchants/:merchant_id/items)" do
+      describe "I see a list of the names of all of my items" do
+        it "And I do not see items for any other merchant" do
+          merchant_1 = Merchant.create!(id: 1, name: "Dani")
```

Comment:
You have an `id` param on one of these merchant creations and not the other. I think the primary keys (id's) will auto-generate if you leave it out.